### PR TITLE
Update package template.

### PR DESCRIPTION
**Pull request by robelgeda**
Working progress... Currently using this PR to test changes incrementally. Will update this description once ready for review.

### CODE REVIEW

**Comment by robelgeda** [comment]
`build_sphinx` passed. Moving on to next commit...

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (C) 2010-2018 Association of Universities for Research in Astronomy (AURA)
+Copyright (c) 2018, Space Telescope Science Institute, AURA
```
Comment (mperrin) [inline LICENSE.md]: I think this part should be "2010-2018" for the copyright period, yes? 
```suggestion
Copyright (c) 2010-2018, Space Telescope Science Institute, AURA
```

**Comment by mperrin** [comment]
Hi @robelgeda, these updates all look pretty straightforward, and there's been no update to this PR in the last couple weeks. Is this good to go? or should I stand by for some additional updates before merging this?

**Comment by robelgeda** [comment]
@mperrin @shanosborne : I've been working on the JDADF projects non-stop (delivery date fast approaching). I am planning on spending all of this week to complete this PR and the other items we discussed. If the end of this week is ok, I can wrap this PR up with time to review it (this also applies to the POOPY repo).

**Comment by mperrin** [comment]
Sure, end of this week is fine. But feel free to use your judgement in prioritizing time between JDADF and this; it's fine from my perspective to do a partial job of this PR for now, and come back and address more of it later after other deadlines for JDADF etc.

**Comment by robelgeda** [comment]
Sounds good, thank you!

**Comment by robelgeda** [comment]
@mperrin @shanosborne : I think this PR is ready to merge. I am going to open another PR for the documentation since it is a significant change in itself.

**Pull request merged**